### DISCUSSION TITLE
More information about how connection works

### DISCUSSION THREAD

**Discussion by julisana** [LeoKlaus/plappa#229] [Q&A] [answered]
I'm looking for more information about how Plappa connects to Jellyfin & ABS. I have a server in my house that is 100% inaccessible from the outside. It has instances of both Jellyfin and ABS on it that work great in the browser. I can connect to them in my browser on my phone/laptop/etc all day long when I'm at home.

When I install Plappa and try and connect to either application (again, while I'm at home and on the same network as the server), I get "Couldn't connect to server: The Internet connection appears to be offline."

If more information can be provided about how Plappa tries to connect locally, that would help me narrow down whether or not there's a setting within my network that can be tweaked or if Plappa is actually having an issue.

**Comment by julisana** [comment, marked as answer]
I ended up figuring out the issue. It was an iOS issue, not anything specific to Plappa or my network.

The issue was that Plappa did not have access within my phone to communicate with devices on my local network.

Once I enabled that, I was able to connect immediately.

**Reply by LeoKlaus** [reply]
Hi Lisa, thank you for the update!

"The internet connection appears to be offline" is the error that's thrown when local network access is disabled.

At least in previous versions of iOS, there was a bug causing this error to pop up even after enabling local network access until the phone was rebooted.


In case anybody stumbles across this in the future:
Plappa only only makes http requests to the server you provided in the setup using the protocol you chose.

It uses the default iOS networking, so settings for that are also respected (VPNs and custom DNS servers work and root certificates you install are also trusted).
